x-pack/plugins/observability_solution/observability_ai_assistant/server/service/client/index.test.ts

```diff
@@ -112,7 +112,7 @@ describe('Observability AI Assistant client', () => {
 
   const knowledgeBaseServiceMock: DeeplyMockedKeys<KnowledgeBaseService> = {
     recall: jest.fn(),
-    getUserInstructions: jest.fn(),
+    getUserInstructions: jest.fn(), // TODO: change
```

Comment:
change to what?

Reply:
Just a note for myself because I was expecting tests to fail here after `getUserInstructions` was changed